### PR TITLE
Add gateway account financial-settings operations

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: openapi/components/schemas/GatewayAccounts/FinancialSettings.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+              allOf:
+                - $ref: ../FilterString.yaml
+            settlementCurrency:
+              $ref: ../CurrencyCode.yaml
+            settlementPeriod:
+              $ref: SettlementPeriod.yaml
+          example:
+            filter: 'currency:EUR,GBP'
+            settlementCurrency: "EUR"
+            period: null
```
Comment (iamdevice) [inline openapi/components/schemas/GatewayAccounts/FinancialSettings.yaml:56]:
```suggestion
            settlementPeriod: null
```
?

**Review by iamdevice — DISMISSED** [review]

**Review by justnero — DISMISSED** [review]

**Comment by slavcoclone** [comment]
@Rebilly/php please re-review

**Comment by slavcoclone** [comment]
@Rebilly/php please review

**Comment by slavcoclone** [comment]
@Rebilly/js please review

**Review by viceromag — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**